install-instructions.php

```diff
@@ -249,8 +249,7 @@
 									<a href="https://owncloud.com/download/?ref=orgB" target="_blank" class="button orange primary mb30">Test Enterprise Edition free for 30 days</a>
 									<hr />
 									<div class="download-notes">
- 										<p><strong>NOTE</strong>: The appliances have been updated to the <a href="https://owncloud.org/blog/new-in-owncloud-9-1/">freshly released ownCloud 9.1</a>. </p>
-										<p><strong>NOTE</strong>: The appliances have been updated to the <a href="https://owncloud.org/blog/owncloud-server-9-0-released/">freshly released ownCloud 9.0</a> but have only gone through absolutely minimal testing. It is recommended to not use these for important data yet - giving them a trial run however and giving us feedback is greatly appreciated!</p>
+ 										<p><strong>NOTE</strong>: The appliances have been updated to <a href="https://owncloud.org/blog/introducing-owncloud-x/">ownCloud 10.0</a> but have only gone through absolutely minimal testing. It is recommended to not use these for important data yet - giving them a trial run however and giving us feedback is greatly appreciated!</p>
```

Comment:
So it looks like the appliances have been tested enough? then we don't need the whole note at all. A mere "The appliances have been updated to the freshly released..." is not needed there. I would then remove the note and find a way to state which exact version we are offering for download above.